DeploymentCloud/Deployment.Common/deployResources.ps1

```diff
@@ -807,12 +828,17 @@ if($resourceCreation -eq 'y') {
 }
 
 if($sparkCreation -eq 'y') {
-    Write-Host -ForegroundColor Green "Deploying resources (2/16 steps): A HDInsight cluster will be deployed"
+    Write-Host -ForegroundColor Green "Deploying resources (2/16 steps): A spark cluster will be deployed"
     Write-Host -ForegroundColor Green "Estimated time to complete: 20 mins"
```

Comment:
should we update this comment? since it wouldn't take 20 mins for databricks,

Reply:
yes

Reply:
done